### PR TITLE
CHECK-168 -- BE: Remove unnecessary exceptions

### PR DESCRIPTION
**Pull request by ChanM0**
# Description

Please include a summary of the change and which issue is fixed. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/DomainValueObjects/Location/Location.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ class Location
   private $address = null;
 
   public function __construct(
-    UUID $uuid = null,
+    string $uuid = null,
```
Comment (JayJayAquino) [inline app/DomainValueObjects/Location/Location.php:21]: why are we no longer type-hinting UUID?

File: app/DomainValueObjects/Location/Location.php [hunk 2]
```diff
@@ -27,7 +27,7 @@ public function __construct(
   }
 
   public static function withAddress(
-    UUID $uuid = null,
+    string $uuid = null,
```
Comment (JayJayAquino) [inline app/DomainValueObjects/Location/Location.php:30]: see above

File: app/Services/RegisterAdminService.php [hunk 3]
```diff
@@ -18,54 +20,58 @@
...
+        'id' => $organizationId,
+        'organization_name' => $organizationName,
         'address' => $address->__toString(),
         'logo' => $logo
       ]);
+      OrganizationSetting::create([
+        'organization_id' => $organizationId
+      ]);
     } catch (\Exception $e) {
+      dd($e);
```
Comment (JayJayAquino) [inline app/Services/RegisterAdminService.php]: why we data dumping

File: database/factories/ProgramFactory.php [hunk 4]
```diff
@@ -8,8 +8,7 @@
 use App\DomainValueObjects\UUIDGenerator\UUID;
 
 $factory->define(Program::class, function (Faker $faker) {
-    $uuid = new UUID('location');
-    $id = $uuid->toString;
+    $uuid = UUID::generate();
     $organizationId = Organization::first();
     $name = "backend";
```
Comment (JayJayAquino) [inline database/factories/ProgramFactory.php]: why are we hard-coding backend

**Review by JayJayAquino — CHANGES_REQUESTED** [review]

File: app/DomainValueObjects/Location/Location.php (hunk 1 above)
Comment (404tonynotfound) [reply app/DomainValueObjects/Location/Location.php:21]: +1

File: app/Exceptions/AuthExceptions/UnauthorizedUser.php [hunk 5]
```diff
@@ -1,14 +1,17 @@
 <?php
 
-namespace App\Exceptions\UserPorfileExceptions;
```
Comment (404tonynotfound) [inline app/Exceptions/AuthExceptions/UnauthorizedUser.php]: just want to point out porfile lol

**Review by 404tonynotfound — APPROVED** [review]

File: app/DomainValueObjects/Location/Location.php (hunk 1 above)
Comment (ChanM0) [reply app/DomainValueObjects/Location/Location.php:21]: cause its unnecessary

**Review by chippingcolt — APPROVED** [review]

**Review by JayJayAquino — APPROVED** [review]
:+1:

**Pull request merged**